cmd/configure_test.go

```diff
@@ -1,82 +1,100 @@
 package cmd
 
 import (
+	"os"
+	"runtime"
 	"testing"
 
 	"github.com/exercism/cli/config"
 	"github.com/stretchr/testify/assert"
 )
 
+type testDefinition struct {
```

Comment:
The changes for breaking out the tests into sub-tests should be broken into a separate PR so that we can review it separately and merge it independently of the Windows fixes.